### ISSUE TITLE
Bug in xSQLServerAvailabilityGroupListener when trying to remove a listener in one case

### ISSUE THREAD

**Issue by johlju** [PowerShell/xSQLServer#96]
If a user want to remove a listener setting `Ensure = 'Absent'` but also adds the port, IP address and DHCP equal to whats already configured, then Test-method will return $true and will not remove the listener.

```
Ensure = 'Absent'
IpAddress = '192.168.10.45/255.255.252.0'
Port = 5030
DHCP = $false
```

_This was found during writing of test for the resource_

**Comment by johlju** [comment]
I am on this. This will be fixed when I send in the PR with the tests.

**Comment by johlju** [comment]
This will be fixed when PR #98 is merged.

**Comment by johlju** [comment]
This was fixed in merged PR #98. Closing.
